meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainer.sh

```
@@ -1,80 +1,123 @@
-   mount /dev/sda1 /tmp/updateFlag
    if [ -f /tmp/updateFlag/JSDELIVR.UPD ]; then
        echo "UPDATE Flag found!" > /dev/tty4
-       echo timer  > /sys/class/leds/nanopi\:blue\:status/trigger
-       echo 50 >   /sys/class/leds/nanopi\:blue\:status/delay_on
-       echo 50 >   /sys/class/leds/nanopi\:blue\:status/delay_off
-
+       echo timer  > /sys/class/leds/user_led/trigger
+       echo 50 >   /sys/class/leds/user_led/delay_on
+       echo 50 >   /sys/class/leds/user_led/delay_off
+
+       docker stop $(docker ps -a -q)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)